macros/remove_prefix_from_columns.sql

```diff
@@ -0,0 +1,12 @@
+{% macro remove_prefix_from_columns(columns, prefix='property_', exclude=[]) %}
```

Comment:
For this general package, I would remove the default prefix parameter value. 
```suggestion
{% macro remove_prefix_from_columns(columns, prefix='', exclude=[]) %}
```

Reply:
sounds good -- i'll change it here and add `prefix='property_'` to any references to this macro (it's just used in the hubspot src package i believe)

Reply:
FYI this will have implications for the current draft of the hubspot PRs. When using this macro, you will need to explicitly define the prefix value.

Reply:
(whoops, did not see your previous comment!) You're on it 💪